src/raz_meta.rs

```diff
@@ -36,27 +36,41 @@ pub trait FirstLast {
 }
 
 impl<E> RazMeta<E> for () {
```

Comment:
Yes, I agree that a standard is useful. I actually added a `fn focus_left()` for this purpose before realizing that the index parameters need first and last as well. See note on other comment.